cartoframes/data/observatory/catalog.py

```diff
@@ -0,0 +1,13 @@
+from cartoframes.data.observatory.category import Categories
+from cartoframes.data.observatory.country import Countries
+
+
+class Catalog(object):
+
+    @property
+    def countries(self):
+        return Countries.get_all()
```

Comment:
I tried to have the properties initialized in the __ init __ method but then I found that mocks were not working in the tests and it tried to execute the real queries (maybe mocks don't work in init time??). This was fixed with this alternative.

Reply:
Since we are reviewing our test system, please, create an issue to fix that in other moment, and link this comment to take it into account.

Reply:
This changes in the next issue (we don't have properties anymore), so no need to a dedicated issue for that. But it would be nice to look more into the behavior and restrictions of Mock/patch for the future.